app/helpers/concerns/foreman_salt/layout_helper_extensions.rb

```diff
@@ -1,20 +0,0 @@
-module ForemanSalt
-  module LayoutHelperExtensions
-    extend ActiveSupport::Concern
-
-    included do
-      alias_method_chain :authorized_associations_permission_name, :salt
```

Comment:
Yea, foreman_salt touches a lot of stuff and tends to break between major releases.  It'll be working in time for the RC, hopefully.  

3.x should work on foreman 1.9 stable.